src/app/application.cpp

```diff
@@ -278,7 +279,7 @@ void Application::torrentFinished(BitTorrent::TorrentHandle *const torrent)
         program.replace("%T", torrent->currentTracker());
         program.replace("%I", torrent->hash());
 
-        QProcess::startDetached(program);
+        std::thread ([program]() { return system(program.toStdString().c_str()); }).detach();
```

Comment:
>But system() does not return until the calling process ends, thus your (detached) thread will not end too.

Maybe it not responsible to say this: If a user want to call a program which will live long, then its up to the user to make sure he calls a daemon (passing parameters). Or he runs `sh myscript.sh`.

You don't put a series of commands in that textbox.